docs/releases/updating_ez_platform.md

```diff
@@ -25,84 +25,66 @@ git checkout -b <branch_name>
-
-If you get a **lot** of conflicts (on the `doc` folder for instance), and eZ Platform was installed from the [share.ez.no](http://share.ez.no) tarball, it might be because of incomplete history. You will have to run `git fetch ezplatform --unshallow` (or `git fetch ezplatform-ee --unshallow`) to load the full history, and run the merge again.
-
-**From your new update branch**
-
-``` bash
-git checkout --theirs composer.lock && git add composer.lock
-```
-
-If you do not keep a copy in the branch, you may also run:
 
-**From your new update branch**
+**1.4.** Inject the changes from the new eZ Platform version (temporary folder) to your branch created to update your project.
 
 ``` bash
-git rm composer.lock
+rsync -avzc --exclude 'src/' --exclude 'web/var' --exclude="YOURSPECIFICS" --exclude=".idea" --delete /tmp/ezplatform <PATH_TO_YOUR_EZPLATFORM_FOLDER>
 ```
 
-## 2. Merge composer.json
-
-#### Manual merging
+!!! tip
 
-Conflicts in `composer.json` need to be fixed manually. If you're not familiar with the diff output, you may checkout the tag's version and inspect the changes. It should be readable for most:
+    The list of excludes is really depending on your project specifics and should be adapted. Ex: if you have eZ in a subfolder it would be
+    --exclude 'ezplatform/src/' --exclude 'ezplatform/web/var' etc. etc.
 
-**From your new update branch**
 
-``` bash
-git checkout --theirs composer.json && git diff HEAD composer.json
-```
+## 2. Check your branch before to update to the new version
 
-You should see what was changed, as compared to your own version, in the diff output. The update changes the requirements for all of the `ezsystems/` packages. Those changes should be left untouched. All of the other changes will be removals of what you added for your own project. Use `git checkout -p` to selectively cancel those changes:
-
-``` bash
-git checkout -p composer.json
-```
+At this stage your project will be updated with the last changes coming from the new eZ Platform version. This procedure
+will remove the custom configuration you have for your project specific.
 
-Answer `no` (do not discard) to the requirement changes of `ezsystems` dependencies. Answer `yes` (discard) to removals of your changes.
+No reason to worry, `git` will help, depending on you organization and your specifics you may want to check the working branch.
 
-Once you are done, inspect the file, either using an editor or by running `git diff composer.json`. You may also test the file's sanity with `composer validate`, and test the dependencies by running `composer update --dry-run`. (will output what it would do to dependencies, without applying the changes.
+ ``` bash
+ git status
+ ```
 
-Once finished, run `git add composer.json` and commit`.`
+ !!! tip
 
-#### Fixing other conflicts (if any)
+     Or using PHPStorm you could do Cmd+K to see the `diff`
```

Comment:
Cmd+K = Mac OS only. You could use the command's name, and provide the Mac OS specific shortcut.